### ISSUE TITLE
combineSelectors:false should not break apart rules

### ISSUE THREAD

**Issue by smirea** [krasimir/absurd#45]
Ideally you would want when running `combineSelectors:true` for absurd to output the rules exactly as you define them, but instead it breaks them apart.

Lets take for example twitter bootstrap's `.clearfix` definition in absurd:

``` javascript
css['.clearfix'] = {
  '*zoom': 1,
  '&:before, &:after': {
    display: 'table',
    content: '',
    lineHeight: 0,
  },
  '&:after': {
    clear: 'both',
  },
};
```

Using `combineSelectors:true` everything is ok:

``` css
.clearfix {
  *zoom: 1;
}
.clearfix:before, .clearfix:after {
  display: table;
  content: "";
  line-height: 0;
}
.clearfix:after {
  clear: both;
}
```

Using `combineSelectors:false` you would expect that the rule `'&:before, &:after': {}` would also be computed in one block, but instead it gets split up (as in, it acts as an "force" don't combineSelectors)

``` css
.clearfix {
  *zoom: 1;
}
.clearfix:before {
  display: table;
  content: "";
  line-height: 0;
}
.clearfix:after {
  display: table;
  content: "";
  line-height: 0;
  clear: both;
}
```

It would be very useful if the combination only happen when requested and otherwise the structure you put in to be compiled in the same format, it helps with testing and debugging a lot more.

In other words, I would have expected that in this particular example, the output from `combineSelectors:true` and `combineSelectors:false` to be the same given that my structure already combines the selectors.
